docs/core-concepts/packages.md

```diff
@@ -0,0 +1,143 @@
+Packages, Metapackages and Frameworks
+=====================================
+
+.NET Core is a platform of NuGet packages. Some product experiences benefit from fine-grained definition of packages while others from course-grained. To accomodate this duality, the product is distributed as a fine-grained set of packages and then described in courser chunks with a package type informally called "metapackage".
+
+.NET Core packages each support being run on multiple .NET runtimes, represented as target frameworks. Some of those frameworks are traditional frameworks, like `net46`, representing the .NET Framework 4.6. Another set is new frameworks that can be thought of as "package-based frameworks", which establish a new model for defining frameworks. These package-based frameworks are entirely formed and defined as packages, forming a strong relationship between packages and frameworks.
+
+Packages
+========
+
+.NET Core is split into a set of packages, which provide primitives, higher-level data types, app composition types and common utilities. Each of these packages represent a single assembly of the same name (eponymous naming). For example, [System.Runtime](https://www.nuget.org/packages/System.Runtime) contains System.Runtime.dll. 
```

Comment:
Do you need to call out the kind of naming here? I had to look it up what eponymous meant, even though I understood your explanation earlier of how the packages were named.